### PR TITLE
reference changed for introducing meshery

### LINKED ISSUE #3023: Docs: Getting error 404 for overview.
#### Current State
I clicked "Introducing Meshery"(https://docs.meshery.io/ ) in the QuickStart section and I'm getting error 404.
https://docs.meshery.io/overview
WhatsApp Image 2021-05-25 at 7 42 59 PM

#### Desired State
This page should open up.
WhatsApp Image 2021-05-25 at 7 43 18 PM

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### LINKED ISSUE #3025: [Docs] Clicking on "Project and Community" opens the "Overview" page.
#### Current State
When a user clicks on the "Project and Community", "Overview" page shows up while the expected page is https://deploy-preview-2090--meshery-docs.netlify.app/project
WhatsApp Image 2021-05-26 at 1 58 17 PM

#### Desired State
The expected page is below.
WhatsApp Image 2021-05-26 at 1 58 33 PM

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Anushka-shukla**
**Description**
Clicking on "Introducing Meshery"(https://docs.meshery.io/ ) in the QuickStart section, error 404 was showing up.
https://docs.meshery.io/overview
This PR,
-Fixes #3023 
-Fixes #3025

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ display-title: "false"
         <div class="overview">Quick&nbsp;Start</div>
     </a>
     <ul><b>Getting Started</b>
-        <li><a href="{{ site.baseurl }}/overview">Introducing Meshery</a></li>
+        <li><a href="{{ site.baseurl }}/project">Introducing Meshery</a></li>
```
Comment (adithyaakrishna) [inline docs/pages/index.md:26]: @Anushka-shukla Could you change it to `/getting -started/overview`
Also, this is not the only file to be changed, do refer this comment https://github.com/layer5io/meshery/issues/3023#issuecomment-848069286

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]

**Comment by adithyaakrishna** [comment]
@Anushka-shukla Would you mind changing the PR description from  
image

to 
This PR,
- Fixes #3023 
- Fixes #3025 

This will link both the issue to the PR

File: docs/pages/getting-started/overview.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-<!-- ---
+
 layout: default
 title: Overview
-permalink: overview
---- -->
```
Comment (adithyaakrishna) [inline docs/pages/getting-started/overview.md:5]: --- -->

Only remove `-->` the other three --- should be present

File: docs/pages/getting-started/overview.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-<!-- ---
```
Comment (adithyaakrishna) [inline docs/pages/getting-started/overview.md:1]: Here also,
<--- ---

Remove only `<---` and the other 3 --- should exist

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]

File: docs/pages/index.md (hunk 1 above)
Comment (Anushka-shukla) [reply docs/pages/index.md]: done

File: docs/pages/getting-started/overview.md (hunk 2 above)
Comment (Anushka-shukla) [reply docs/pages/getting-started/overview.md:5]: done

File: docs/pages/getting-started/overview.md (hunk 3 above)
Comment (Anushka-shukla) [reply docs/pages/getting-started/overview.md:1]: done

**Review by adithyaakrishna — APPROVED** [review]

**Comment by adithyaakrishna** [comment]
LGTM 🎉
Nice Work @Anushka-shukla 🚀

**Comment by leecalcote** [comment]
Good teamwork. 👍

**Comment by leecalcote** [comment]
Yay, @Anushka-shukla! 🎈

**Pull request merged**